gallery/src/components/NxToast/NxToastComplexLayoutExample.tsx

```diff
@@ -39,15 +39,19 @@ const sidebarLogoPath = require('../../assets/images/logo-plaid-villain-text.png
 
 interface ToastModel {
   id: number;
-  alertComponent: ComponentType;
+  alertComponent: ComponentType<alertComponentProps>;
   message: string;
 }
 
+type alertComponentProps = {
```

Comment:
Type names should use PascalCase (e.g. start with an uppercase letter)

Reply:
> Type names should use PascalCase (e.g. start with an uppercase letter)

Updated here: [b746450](https://github.com/sonatype/sonatype-react-shared-components/pull/766/commits/b7464505042e11a619390940815b23e953f069d8)